backend/routes.js

```diff
@@ -250,6 +251,8 @@ const setup = ({
 
   router.get('/back-to-start', backToStart())
 
+  router.use('/appointment-details/:id', appointmentDetails({ prisonApi, whereaboutsApi }))
```

Comment:
For later - maybe use a dedicated router for appointments - we are going to have a few pages.